src/pages/restaurant/components/dishItems/DishItems.js

```diff
@@ -1,32 +1,100 @@
 // react
-import React, { useMemo } from "react";
+import React, { useMemo, useState } from "react";
 
 //prop-types
 import PropTypes from "prop-types";
 
 //lodash
 import _keys from "lodash/keys";
+import _isEmpty from "lodash/isEmpty";
+
+// helpers
+import {
+  filterItemIfQuantityIsZero,
+  findItemById,
+  updateItemQuantity,
+} from "./helpers/dishItems.general";
+import { saveCartItems } from "../../../../utility/localStorage";
 
 // components
 import Sidebar from "../../../../commonComponents/sidebar";
 import Menu from "./components/menu";
+import IconWithLabel from "../../../../commonComponents/iconWithLabel";
+import Cart from "../../../../commonComponents/cart";
+
+// icons
+import cartEmpty from "../../../../assets/images/cart_empty.png";
 
 // css
 import "./dishItems.css";
+import { compose } from "../../../../utility/compose";
+
+const DishItems = ({ dishItems }) => {
+  const [cart, setCart] = useState({
+    items: [],
```

Comment:
You can directly store the items right why do you need to create and object and keep items in that

Reply:
done